llvm/lib/CodeGen/SelectionDAG/LegalizeVectorTypes.cpp

```diff
@@ -3534,11 +3550,9 @@ SDValue DAGTypeLegalizer::SplitVecOp_EXTRACT_VECTOR_ELT(SDNode *N) {
   Align SmallestAlign = DAG.getReducedAlign(VecVT, /*UseABI=*/false);
```

Comment:
Can we compare with `DAG.getSubtarget().getFrameLowering()->getStackAlign()` and use the smaller one here?

Reply:
Added!